Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -221,7 +221,7 @@ function IsPrBuild
 
 function GetSonarQubeBuildDirectory
 {
-    $agentBuildDirectory = GetTaskContextVariable "Agent.BuildDirectory" 
+    $agentBuildDirectory = $env:Agent_BuildDirectory 
```

Comment:
Is this recommendation documented somewhere, and are there any others we should be following?